mlflow/telemetry/constant.py

```diff
@@ -0,0 +1,6 @@
+# TODO: update this url with a custom domain
+TELEMETRY_URL = "https://twqhrx9tai.execute-api.us-west-2.amazonaws.com/test/telemetry"
```

Comment:
what would happen if an attacker sent 1B requests to this endpoint now?

Reply:
Good point :) I'll delete this and use a different one for testing